integration-tests/src/test/java/org/apache/fineract/integrationtests/BatchApiTest.java

```diff
@@ -113,12 +110,14 @@ public void setup() {
         this.requestSpec.header("Authorization", "Basic " + Utils.loginIntoServerAndGetBase64EncodedAuthenticationKey());
         this.responseSpec = new ResponseSpecBuilder().expectStatusCode(200).build();
         this.datatableHelper = new DatatableHelper(this.requestSpec, this.responseSpec);
-        GlobalConfigurationHelper.updateIsAutomaticExternalIdGenerationEnabled(this.requestSpec, this.responseSpec, true);
+        globalConfigurationHelper.updateGlobalConfiguration("enable-auto-generated-external-id",
```

Comment:
I think these names should go into a class and have these as constants. Can't we do it in fineract-provider and use it from there?